app/data_model/answer_store.py

```diff
@@ -122,6 +122,20 @@ def add_or_update(self, answer):
         except ValueError:
             self.add(answer)
 
+    def add_multiple_answers(self, answers):
+        """
+        Add a new answers into the answer store, or update if it exists.
```

Comment:
typo get rid of a

Reply:
you mean the `a` yeh?

Reply:
yes srry method is multiple answers, my bad